disasterinfosite/templates/index.html

```diff
@@ -24,13 +28,14 @@
   <!-- Load Google Maps from CDN-->
   <script src="https://maps.googleapis.com/maps/api/js?v=3.exp"></script>
 
-  <script src="{% static 'js/vendor/modernizr.js' %}"></script>
-  <script src="{% static 'js/vendor/jquery.js' %}"></script>
-  <script src="{% static 'js/foundation.min.js' %}"></script>
-  <script type="text/javascript">
-    var mapBounds = {{ data_bounds | js }};
-  </script>
-  <script src="{% static 'js/app.js' %}"></script>
+  <!-- Load Slick from CDN -->
+<link rel="stylesheet" type="text/css" href="//cdn.jsdelivr.net/jquery.slick/1.5.9/slick.css"/>
+<script type="text/javascript" src="//cdn.jsdelivr.net/jquery.slick/1.5.9/slick.min.js"></script>
```

Comment:
I had problems with loading slick locally, and I don't know if everything else is available from a CDN (but I'd be happy to load it from one if it is)